src/main/groovy/com/jetbrains/python/envs/PythonEnvsExtension.groovy

```diff
@@ -125,8 +147,22 @@ class PythonEnvsExtension {
         )
     }
 
+    void textfile(final File file, final String content) {
+        files << new CreateFile(file, content)
+    }
+
     void textfile(final String path, final String content) {
-        files << new CreateFile(path, content)
+        textfile(new File(envsDirectory, path), content)
+    }
+
+    void link(final Path link, final Path source) {
+        links << new CreateLink(link, source)
+    }
+
+    void link(final String linkString, final String sourceString) {
+        Path linkPath = new File(envsDirectory, linkString).toPath()
```

Comment:
Good point, thanks! 
I've fixed link and textfile methods to accept String arguments. But in link method I also have added baseDir argument for not to duplicate beginnings of link and source paths.